src/models/site/useGetTwitterbotTestResults.js

```diff
@@ -0,0 +1,12 @@
+import queryKeys from '../../constants/queryKeys';
+import useFetch from '../../hooks/useFetch';
+
+export default function useGetTwitterbotTestResults() {
+  return useFetch({
+    queryKey: queryKeys.twitterBot,
```

Comment:
where possible let's name query keys after the data that is returned. so not twitterBot here, but twitterBotTestResult

Reply:
Done